builtins-default-costs/src/lib.rs

```diff
@@ -1,4 +1,5 @@
 #![cfg_attr(feature = "frozen-abi", feature(min_specialization))]
+#![feature(const_option)]
```

Comment:
this is for `const fn validate_position()` to compare `position` that's inside Option<>. Probably not acceptable by release build. Would be great to validate `position` at compile time, considered to move `position` outside of Option, but that made `NON_MIGRATING_BUILTINS_COSTS` look weird since `position` is not required for those.

Reply:
yep, CI isn't happy. Going to make one type w `position` for migrating-builtins, and another type without for not-migrating-builtins.

Reply:
e3ded012f80510672ee17bee192dc7f5e3398167